src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -164,9 +170,29 @@ async function getQuote(
 ): Promise<CoverRouterQuoteResponse> {
   const params: CoverRouterQuoteParams = { productId, amount: coverAmount, period: coverPeriod, coverAsset };
   const response = await axios.get<CoverRouterQuoteResponse>(process.env.COVER_ROUTER_URL + '/quote', { params });
+  if (!response.data) {
+    throw new Error('Failed to fetch cover quote');
```

Comment:
Can we add the original error here as a `cause` prop to the second param of the `Error` constructor? Might be nice for debugging some day

Reply:
This is just an error handler if the API for some reason does not return a response data. So there is no original error, the error is "Failed to fetch cover quote"